ovirtlib4/vms.py

```diff
@@ -63,7 +63,7 @@ def disks(self):
     def backups(self):
         return VmBackups(connection=self.service)
 
-    def start_and_wait(self, state='up', wait_timeout=defaults.VM_START_TIMEOUT, *args, **kwargs):
+    def start_and_wait(self, state=types.VmStatus.UP.value, wait_timeout=defaults.VM_START_TIMEOUT, *args, **kwargs):
```

Comment:
```suggestion
    def start(self, state=types.VmStatus.UP.value, wait_timeout=defaults.VM_START_TIMEOUT, *args, **kwargs):
```
I would actually leave the 'wait' out of the method name and keep it as an optional parameter.
What the method does is to start a VM. I think the waiting should be optional.
If any I would keep the start simple enough, and create another method which waits for the VM to be in a certain state

Reply:
If you want to start a VM then you should use the build in API method vm.service.start() 
but start_and_wait does not implemented

Reply:
Very well